env.local.sh

```
@@ -0,0 +1,12 @@
+export AKENEO_BASE_URL="https://hcl-9095c85a9e.trial.akeneo.cloud/"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)